UI/Web/src/app/person-detail/person-detail.component.scss

```diff
@@ -1,4 +1,19 @@
 .main-container {
   margin-top: 10px;
   padding: 0 0 0 10px;
+
+  .person-badge {
```

Comment:
Last I saw on the image component, this wont take effect due to the view encapsulation. Can you validate for me?

Reply:
Seems to be working for me on the dev server:

![image](https://github.com/user-attachments/assets/1ebb6253-3897-461d-813d-88414910ccbf)

![image](https://github.com/user-attachments/assets/99b5925a-f0b5-4261-a458-efaa9dafb4cc)